.gitattributes

```diff
@@ -1 +1,2 @@
 *.html linguist-language=python
+* text=auto eol=lf
```

Comment:
This was added to force git to use lf line breaks. Since I am on windows everything keeps want to change it to crlf. I hope this is OK?